pom.xml

```diff
@@ -63,14 +63,16 @@
         <dep.testcontainers.version>1.17.6</dep.testcontainers.version>
         <dep.duct-tape.version>1.0.8</dep.duct-tape.version>
         <dep.coral.version>2.0.77</dep.coral.version>
-        <dep.confluent.version>5.5.2</dep.confluent.version>
+        <dep.confluent.version>7.3.1</dep.confluent.version>
+        <dep.kafka-clients.version>3.3.2</dep.kafka-clients.version>
         <dep.casandra.version>4.14.0</dep.casandra.version>
         <dep.minio.version>7.1.4</dep.minio.version>
         <dep.iceberg.version>1.1.0</dep.iceberg.version>
         <dep.spotbugs-annotations.version>4.7.2</dep.spotbugs-annotations.version>
         <dep.protobuf.version>3.21.6</dep.protobuf.version>
-        <dep.wire.version>3.2.2</dep.wire.version>
-        <dep.kotlin.version>1.4.0</dep.kotlin.version>
+        <dep.wire.version>4.4.3</dep.wire.version>
```

Comment:
Can we switch to importing `wire-bom` as a `<scope>import</scope>` and remove this property?

Reply:
We cannot. bom doesn't contain schema-jvm, runtime-jvm libraries: https://repo1.maven.org/maven2/com/squareup/wire/wire-bom/4.5.3/wire-bom-4.5.3.pom